WoWExportTools/Loaders/ADTLoader.cs

```diff
@@ -328,7 +309,12 @@ public static Terrain LoadADT(Structs.MapTile mapTile, int shaderProgram, bool l
             result.doodads = doodads.ToArray();
             result.worldModelBatches = worldModelBatches.ToArray();
 
-            tileCache.Add(mapTileKey, result);
+            // Clean-up.
+            foreach (var batch in renderBatches)
```

Comment:
@Marlamin Is this safe to do here? This was something done when "clearing" the ADT terrain cache, so I brought it over after removing said caching. Unsure if it needs to be done later or if this is fine.